services/agent/src/simcore_service_agent/modules/volumes_cleanup/_core.py

```diff
@@ -0,0 +1,65 @@
+import logging
+
+from ...core.settings import ApplicationSettings
+from ._docker import delete_volume, docker_client, get_dyv_volumes, is_volume_used
+from ._s3 import store_to_s3
+
+logger = logging.getLogger(__name__)
+
+
+async def backup_and_remove_volumes(settings: ApplicationSettings) -> None:
+    async with docker_client() as client:
+        dyv_volumes: list[dict] = await get_dyv_volumes(client)
+
+        if len(dyv_volumes) == 0:
```

Comment:
MINOR: i think ``if not dyv_volumes`` reads better . but both are prefectly fine ;-)

Reply:
really? not my pylint :-)

```python
def bar() -> list[dict]:
    ...

def foo():
    dyv_volumes: list[dict] = bar()
    if dyv_volumes:
        print("has items")
```

```cmd
pylint ignore.py             

--------------------------------------------------------------------
Your code has been rated at 10.00/10 (previous run: 10.00/10, +0.00)
```